src/pages/responsible-trading/index.js

```diff
@@ -33,10 +41,40 @@ const StyledHeader = styled(Header)`
     }
 `
 
+const CFDWrapper = styled(Text)`
+    background-color: #f9fafc;
```

Comment:
please check the color in variables.js, if its not available, please add new color variable

Reply:
done